javascript-light-bulb/styles.css

```diff
@@ -0,0 +1,65 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: rgb(17 17 17);
+}
+
+.dark-back-light {
+  background: rgb(17 17 17);
+}
+
+.dim-back-light {
+  background: #faea84;
+}
+
+.bright-back-light {
+  background: white;
+}
+
+.bulb {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  background: rgb(80 80 80);
+  border-radius: 50%;
+}
+
+.bulb-off {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  background: rgb(80 80 80);
+  border-radius: 50%;
+}
+
+.bulb-half-on {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  background: rgb(148 123 11);
+  border-radius: 50%;
+}
+
+.bulb-on {
+  position: relative;
```

Comment:
`position: relative` is not needed unless you were using top, right, left, bottom offset property or needed to set element as parent for a child div set to position absolute.